app/src/features/sessionBook/context/actions.tsx

```diff
@@ -17,6 +17,9 @@ export const SessionsActionContextProvider: React.FC<RulesProviderProps> = ({ ch
   const handleDeleteSessionAction = useCallback((id: string, eventsFilePath: string, handleForceRender: () => void) => {
     Modal.confirm({
       title: "Confirm",
+      /* antD modal by default applied warining and size styles to antd icons
+      TODO: use react-icons instead of antd icons when revamping sessions UI
+      */
```

Comment:
Try using react-icon and give it a class as `anticon`, it should work [see](https://github.com/requestly/requestly/blob/86b74ba60b774d7300301446bfddadab599047bd/app/src/features/incentivization/components/IncentiveTasksList/components/NewRuleButton/NewRuleButton.tsx#L32)